geomesa-hbase/geomesa-hbase-server/src/main/scala/org/locationtech/geomesa/hbase/server/common/CoprocessorScan.scala

```diff
@@ -67,8 +66,10 @@ trait CoprocessorScan extends StrictLogging {
         logger.debug(s"Initializing aggregator $aggregator with options ${options.mkString(", ")}")
         aggregator.init(options)
 
-        val scan = ProtobufUtil.toScan(ClientProtos.Scan.parseFrom(Base64.getDecoder.decode(options(GeoMesaCoprocessor.ScanOpt))))
-        scan.setFilter(FilterList.parseFrom(Base64.getDecoder.decode(options(GeoMesaCoprocessor.FilterOpt))))
+        val scan = {
```

Comment:
Did you break this up on three lines to make it more readable?  Or increase your sloc count?!;)

Reply:
both?? trying to not exceed the row count...